### DISCUSSION TITLE
Textural inversion

### DISCUSSION THREAD

**Discussion by nawnie** [TheLastBen/fast-stable-diffusion#408] [Ideas]
Was wandering if there was anyway to use an embedding file from the ui or cypress' inversion to aide in a dreambooth creation. Since that would already have some of the information used in training the encoder [step1] or a way to write our own text. Or even add more tags to each image, maybe any word after the first becomes a tag to the referenced subject.

**Comment by TheLastBen** [comment]
you can add a tag to images in their filenames, if you use an identifier "abcd", make the images names : abcd_tag1_tag2.jpg ....etc
but you should know that if you add captions or tags to the images, the model becomes overfit and won't be as retrainable.

**Reply by nawnie** [reply]
my current project is to get some photos of my uncle who passed away not long ago so i am trying to work with very little including different ages. and make some memorial art, so this would be a model on its own not to update. thank you so much for replying.
